apps/studio/components/interfaces/Auth/Policies/PolicyTableRow/PolicyTableRow.utils.ts

```diff
@@ -12,7 +12,7 @@ export const generatePolicyCreateSQL = (policy: PostgresPolicy) => {
   }
 
   return `
-CREATE POLICY "${policy.name}" 
+CREATE OR ALTER POLICY "${policy.name}" 
```

Comment:
this is not valid SQL iinw - its either "create policy" or "alter policy", no "create or alter"